tasks/configure.js

```diff
@@ -97,18 +105,23 @@ function task (opts) {
 		});
 	}
 
-	let authorizedPostHeaders = {
-		'Accept': 'application/vnd.heroku+json; version=3',
+	const acceptHeader = target === 'review-app' ? 'application/vnd.heroku+json; version=3.pipelines' : 'application/vnd.heroku+json; version=3';
```

Comment:
Given we have both the URL and the header to change, would it perhaps be better to combine both in the same conditional:

```
let fetchConfig;

if(target === 'review-apps') {
   fetchConfig = fetch( reviewAppUrl, { headers});
} else {
   fetchConfig = fetch( regularUrl, { headers});
}

...

Promise.all([
   blah,
   fetchConfig,
   blah
]);

```

Reply:
(not necessarily an if statement like that - but basically only having one conditional the decides both at the same time.)